Maple2.Server.Game/Manager/QuestManager.cs

```diff
@@ -355,6 +361,53 @@ public bool Complete(Quest quest, bool bypassConditions = false) {
         return true;
     }
 
+    private bool CanHoldQuestRewards(QuestMetadataReward reward) {
```

Comment:
I wonder if this will serve better as a function in InventoryManager? IIRC Trading should also make use of this.